src/IOManagement.cpp

```diff
@@ -0,0 +1,93 @@
+// combined source file of digitalPin.cpp and analogPins.cpp 
+// combined variables / functions given the name hybrid
+
+#include "IOManagement.h"
+
+// from digitalPins.cpp
+#include "const.h"
+#include <Arduino.h>
+#include "STM32TimerInterrupt_Generic.h"
+
+volatile Digital_Data digital_data;
+
+// from analogPins.cpp
+volatile float acc_out = 0;
+volatile float regen_brake = 0;
+volatile float lv_12V_telem = 0;
+volatile float lv_5V_telem = 0;
+volatile float lv_5V_current = 0;
+volatile float current_in_telem = 0;
+volatile float brake_pressure_telem = 0;
+
+// combined timer of digitalReader() and dataUpdater()
+// Ticker to poll input readings at fixed rate
+STM32TimerInterrupt hybridTimer(TIM7); // used TIM7 from digital, analog used TIM2
```

Comment:
The in-line comment isn't necessary because TIM2 and TIM7 both are timers and can be used interchangeably. It's like how the MCU has multiple digital pins.